src/style.css

```diff
@@ -0,0 +1,153 @@
+body {
+  background: #f6f6f6;
+}
+
+body * {
+  color: rgb(84, 88, 98);
+  font-family: 'Open Sans', 'Lucida Grande', tahoma, verdana, arial, sans-serif;
+}
+
+main {
+  justify-content: center;
+  align-items: center;
+  min-height: 95vh;
+  margin: 0 auto;
+}
+
+.todo-list {
+  box-shadow: rgba(0, 0, 0, 0.15) 0 2px 8px 0;
+  width: 500px;
+  max-width: 100%;
+}
+
+form button .material-icons {
+  font-size: 15px;
+}
+
+.todo-list .material-icons {
+  transition: 0.6s;
+  color: #9797a5;
+  cursor: pointer;
+}
+
+.header .material-icons {
+  font-size: 18px;
+}
+
+.todo-list .material-icons:hover {
+  color: #333;
+}
+
+.todo-list .material-icons.move {
+  cursor: move;
+}
+
+.header {
+  border-bottom: 1px solid rgba(0, 0, 0, 0.08);
+}
+
+.header,
+form,
+ul {
+  background: #fff;
+  margin: 0;
+  width: 100%;
+  padding: 0;
+  list-style-type: none;
+}
+
+.header,
+h2 {
+  align-items: center;
+  color: #545862;
+  font-size: 18px;
+  font-weight: 400;
+  padding: 1rem;
+  line-height: 20px;
+  margin: 0;
+  width: calc(100% - 2rem);
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+}
+
+form input {
+  width: calc(100% - 25px);
+  border: 0 solid transparent;
+  outline: none;
```

Comment:
In following CSS best practice always 
Be careful with blank lines and indentation.
Do not add blank lines, spaces, or indentations without a reason.
For readability, add blank lines to separate large or logical code blocks.